static/app/components/feedback/feedbackItem/feedbackScreenshot.tsx

```diff
@@ -22,6 +27,8 @@ export default function FeedbackScreenshot({
   onClick,
 }: Props) {
   const [isLoading, setIsLoading] = useState(true);
+  const isImage = screenshot.mimetype.includes('image');
```

Comment:
based on @c298lee's review I was wonder what could be. Mime type isnt' a required field on envelopes. So might be wroth having a fallback to file extension.

We could have some reusable logic in issue detail, by attachment preview

Reply:
ahh @bruno-garcia the case @c298lee brought up is because the mimetype is `application/png` instead of `image/png` -- maybe a wrong mimetype setting? but i'll make a case for that

Reply:
got it.

What fills out the `mimetype` property? It's not then the value from attachment, is it?
Does it align with Issue details attachment preview?

Reply:
`application/png` seems incorrect to me, did we add that by hand?

I think the file extension will work better with data coming from the wild. Since we expose a `addAttachment` API that doesn't require mimetype let alone users adding the right value

Reply:
yeah @bruno-garcia i'm not totally sure, `application/png` doesn't seem right to me either. because it was from a test feedback, maybe @c298lee knows? maybe we manually set that at some point?

Reply:
i think the issue details preview renderer requires the right type to set, otherwise it returns `undefined1
https://github.com/getsentry/sentry/blob/f2003b5ed3eff14b1481fe99fa154398fa94435a/static/app/components/events/eventAttachments.tsx#L38-L56

Reply:
I believe we should consider the file extension for images since the mine type is added "by hand" and can and is wrong often

Reply:
I agree with Bruno, it could be added "by hand", and the `addAttachment` API defaults to `application/octet-stream` even if it's an image. However I'm not sure how well using file extension for images would work since the name of the file could also be added "by hand"

Reply:
@c298lee @bruno-garcia i figured out a way around casing on the `mimetype` -- instead, i'm using the image error state to tell me whether an image failed or not. we'll attempt to load all attachments as errors first, and if `<img>` gives an error because the `src` it tries to load is not in fact an image, then we tell that to our image renderer and render the default preview instead.